### PR TITLE
[AF-1254]: Examples Import and Project import logic split to improve readability and to remove unwanted behavior

### PR DESCRIPTION
**Pull request by adrielparedes**
Created ProjectImportService to separate behavior from ExampleService. Also importProject from LibraryService moved to ProjectImportService

Related to:
* https://github.com/kiegroup/jbpm-wb/pull/1190

### CODE REVIEW

**Comment by adrielparedes** [comment]
jenkins execute full downstream build please

**Comment by adrielparedes** [comment]
@kkufova There are several changes in the backend that made things behave ugly. One of them is that now every imported project is a git clone with history, you can see it now from Metrics tab. Also, bare git repos generated from import a project or an example changed their names. (example now contain an "example-" prefix). The logic used to be this was written in the same place, and the main goals of this PR was to split that logic in different services.

About Import Projects Screen I think that a git repository may contain several projects to be imported and we still need some UI to choose between those. But as I'm not totally sure about that lets give @ederign the possibility to comment

**Comment by jhrcek** [comment]
@adrielparedes Thanks for the explanation. Since this affects Metrics (and preservation of commit history) I'll need to check this too (and update tests for Project / Space metrics). I'll provide some feedback with the results tomorrow.

File: kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-api/src/main/java/org/kie/workbench/common/screens/examples/validation/ExampleProjectValidator.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
```
[the remark this answers is not in the document]
Comment (adrielparedes) [reply kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-api/src/main/java/org/kie/workbench/common/screens/examples/validation/ExampleProjectValidator.java]: :+1:

**Comment by ederign** [comment]
jenkins retest this

**Pull request merged**